docs/en/serverless/logging/log-monitoring.mdx

```diff
@@ -79,6 +79,13 @@ The following resources provide information on viewing and monitoring your logs:
 - <DocLink slug="/serverless/observability/discover-and-explore-logs">Discover and explore</DocLink>: Discover and explore all of the log events flowing in from your servers, virtual machines, and containers in a centralized view.
 - <DocLink slug="/serverless/observability/aiops-detect-anomalies">Detect log anomalies</DocLink>: Use ((ml)) to detect log anomalies automatically.
 
+## Monitor datasets
+
+The **Datasets** page provides an overview of your datasets and their quality.
```

Comment:
Page is currently called `Data Set Quality`, should we align this?
<img width="857" alt="image" src="https://github.com/elastic/observability-docs/assets/1313018/b2b96707-8c25-4bda-90b5-86e0642d8135">